layouts/partials/integrations-logo.html

```diff
@@ -0,0 +1,105 @@
+<!--
+{{/*
+------------------
+Description:
+------------------
+
+Builds a string to the integration logo.
+:param basename: string of integration name without any variant or extension e.g "aws" or "amazon-ecs"
+:param variant: string of the variant e.g "avatar" or "small" or "large"
+:return: the string to the integration logo image or an empty string on failure
+
+------------------
+Usage:
+------------------
+
+With an exact integration id
+
+{{ $int_logo := partial "integrations-logo.html" (dict "context" $dot "basename" "amazon-alb" "variant" "avatar") }}
+<img src="{{ $int_logo }}" />
+
+With an exact integration alias
+
+{{ $int_logo := partial "integrations-logo.html" (dict "context" $dot "basename" "aws_alb" "variant" "avatar") }}
+<img src="{{ $int_logo }}" />
+
+Both of these should return the same image.
+*/}}
+-->
+{{- $dot := . -}}
+{{- $ctx := .context -}}
+{{- $basename := (lower .basename) -}}
+{{- $variant := .variant | default "large" -}}
+{{- $extension := .extension | default ".svg" -}}
+{{- $img_url := $ctx.Site.Params.integrations_img_url -}}
+{{- $imagesJSON := getJSON "https://s3.amazonaws.com/origin-static-assets/websites-integrations/images.json" -}}
```

Comment:
how is this file generated? what's the process for adding new integrations logos or fixing an existing logo